### DISCUSSION TITLE
Settings question

### DISCUSSION THREAD

**Discussion by ryumaou** [Vaccarini-Lorenzo/iCalObsidianSync#22] [Q&A] [answered]
What's this supposed to show on the Settings tab?
![image](https://github.com/Vaccarini-Lorenzo/iCalObsidianSync/assets/371696/f21cffad-4cfb-475b-89ee-16d8467d11e4)

Is that for selecting a calendar if there's more than one calendar?
Also, I'm using the Checklist, Tasks, and Full Calendar plugins to manage my ToDo lists.  When I add a task to the calendar, how long will it take to sync back to my iCloud calendar?

This plugin will my organization plan/GTD/Second Brain setup really sing!  Thank you!

**Comment by Vaccarini-Lorenzo** [comment, marked as answer]
The 'Calendar' dropdown setting functions exactly as you described: if you have multiple calendars, you can dynamically choose the one you want to work with. 
![Screenshot 2023-09-04 at 21 25 01](https://github.com/Vaccarini-Lorenzo/iCalObsidianSync/assets/80112134/8a1c6cbd-f9f9-40ee-bc60-27672ca1b04c)
This means that you can synchronize different events across different calendars or embed events belonging to a particular calendar in your notes with the new inline event view feature (still a beta).

Regarding synchronization time, it should be seamless (though this depends on your internet connection). As soon as you click the 'Sync' button, a request is sent and your calendar is updated.
Unfortunately, I'm a fairly new Obsidian user and I'm not very familiar with the plugins you mentioned, but if you're using other plugins to interact with your remote calendar, the sync time depends entirely on their internal implementation. 

I'm glad you find this plugin useful! Don't hesitate to open more issues/discussions if something is unclear.

**Comment by ryumaou** [comment]
I don't see a "sync button"  Is that in the plugin somewhere or in iCloud?

**Comment by Vaccarini-Lorenzo** [comment]
I was referring to this "Sync button"
![Screenshot 2023-09-04 at 22 21 56](https://github.com/Vaccarini-Lorenzo/iCalObsidianSync/assets/80112134/69a31e68-5978-4dff-b4a1-a6aed7db61c5)
If your question was about the synchronization time between the inline event view, currently, the visualization is static (it refreshes every time Obsidian renders the code block). In the next releases, the feature will be improved with real-time synchronization and the possibility to interact directly with the events (moving them around, modifying the content/time, etc).

**Comment by ryumaou** [comment]
Ah!  Now, I see!  Thank you for the explanation.  If there's anyway I can support the development of the autosync feature, please, let me know!

**Comment by Vaccarini-Lorenzo** [comment]
Absolutely no problem.
Thank you for your interest, I'll soon add a TODO list to the README file as a reference so that everybody can freely contribute.

**Comment by Vaccarini-Lorenzo** [comment]
Hi there, the 2 way sync feature is finally available for Apple Calendar!
